backup_and_restore/replacing-failed-master.adoc

```diff
@@ -0,0 +1,33 @@
+[id="replacing-failed-master-host"]
+= Replacing a failed master host
+include::modules/common-attributes.adoc[]
+:context: replacing-failed-master-host
+
+toc::[]
+
+This document describes the process to replace a single etcd member. This procedure assumes that there is still an etcd quorum in the cluster.
+
+[NOTE]
+====
+If you have lost the majority of your master hosts, leading to etcd quorum loss, then you must follow the disaster recovery procedure to xref:../backup_and_restore/disaster_recovery/scenario-1-infra-recovery.adoc#dr-infrastructure-recovery[recover from lost master hosts].
+====
+
+To replace a single master host:
+
+* xref:../backup_and_restore/replacing-failed-master.adoc#restore-remove-failed-master_replacing-failed-master-host[Remove the member from the etcd cluster].
+
+* xref:../backup_and_restore/replacing-failed-master.adoc#restore-add-master_replacing-failed-master-host[Add the member back to the etcd cluster].
+
+// TODO: Confirm. But how can you use this to scale up if the procedure assumes that you're adding a member that had previously been removed?
+You can also follow these procedures individually to scale your master hosts up or down as needed.
+
+// Removing a failed master host from the cluster
+include::modules/restore-remove-failed-master.adoc[leveloffset=+1]
+
+// Adding a master host back to the cluster
+include::modules/restore-add-master.adoc[leveloffset=+1]
+
+.Next steps
+
+// TODO: Does this need to be done before or after adding the member back to the cluster?
```

Comment:
@retroflexer If the certs aren't valid, does the DR scenario to recover from expired certs need to be done BEFORE or AFTER adding the member back to the etcd cluster?

Reply:
If the certs aren't valid, we recommend them to use recover script.

`recover` script will take care of adding the certs as well as adding the member back to the etcd cluster.